org.sugarlabs.TypingTurtle.json

```diff
@@ -0,0 +1,268 @@
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://pypi.python.org/packages/70/f1/cb9373195639db13063f55eb06116310ad691e1fd125e6af057734dc44ea/decorator-4.2.1.tar.gz",
+                    "sha256": "7d46dd9f3ea1cf5f06ee0e4e1277ae618cf48dfb10ada7c8427cd46c42702a0e"
+                }
+            ]
+        },
+        {
+            "name": "python-six",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links \"file://${PWD}\" --install-option=\"--prefix=${FLATPAK_DEST}\" six"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/dd/bf/4138e7bfb757de47d1f4b6994648ec67a51efe58fa907c1e11e350cddfca/six-1.12.0.tar.gz",
+                    "sha256": "d16a0141ec1a18405cd4ce8b4613101da75da0e9a7aec5bdd4fa804d0e0eba73"
+                }
+            ]
+        },
+        {
+            "name": "telepathy-glib",
+            "config-opts": ["--disable-static", "--disable-gtk-doc"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://www.github.com/TelepathyIM/telepathy-glib.git",
+                    "tag": "telepathy-glib-0.24.1",
+                    "commit": "c834ef1a50a97cbfae7a0d560ff806fd5aa7ffe8"
+                },
+                {
+                    "type": "patch",
+                    "path": "telepathy-glib-uniquify.patch"
+                }
+            ]
+        },
+        {
+            "name": "sugar-toolkit-gtk3",
+            "no-parallel-make": true,
+            "config-opts": [
+                "--with-python3"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.sugarlabs.org/sources/sucrose/glucose/sugar-toolkit-gtk3/sugar-toolkit-gtk3-0.116.tar.xz",
+                    "sha256": "745a360577efceb10f57ca4cbad5f9ec61f7ed6cc381eeb72422bbd7455b770b"
+                }
+            ]
+        },
+        {
+            "name": "sugarapp",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links ${PWD} --install-option=\"--prefix=${FLATPAK_DEST}\" $PWD"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/tchx84/sugarapp.git",
+                    "tag": "v1.10",
+                    "commit": "54b0efdc0bbef60d48685aa2a1559a3fb8f2ddf5"
+                }
+            ]
+        },
+	{
+	    "name":"myspell",
+	    "buildsystem": "simple",
+	    "build-commands" : [
+            "mkdir /app/share/myspell",
+            "cp *.dic /app/share/myspell/."
+	    ],
+	    "sources" : [
+            {
+                "type": "file",
```

Comment:
You should clone the https://anongit.freedesktop.org/git/libreoffice/dictionaries.git repo and from there copy the files you need. This is important because Flatpak apps are supposed to be reproducible. So, to achieve that we need to specify a commit, so we guarantee that we always use the exact the versions of these files.

It will make your manifest shorter as well.

Reply:
myspell and its dictionaries are removed from the json :)